.gitignore

```diff
@@ -34,3 +34,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.idea/
+.rakeTasks
```

Comment:
You can add the `.generators` file in here as well

Reply:
Done.